data-serving/scripts/export-data/transform.py

```diff
@@ -271,30 +274,54 @@ def open_writers(formats: list[str], fields: list[str], output: str):
         print("Error occurred in open_writers():")
         print(e)
     finally:
+        if output == '-':
+            return
         for fmt in files:
             if fmt == "json":
                 files[fmt].write("]\n")
             files[fmt].close()
```

Comment:
Should this be `for fmt in formats` and `if files[fmt] == "json"`?

Reply:
files[fmt] refers to file objects